### PR TITLE
Standardize docker image name

### PR DESCRIPTION
**Pull request by ar13pit**
- Replace `master` tag with `latest` for default branch as that is the docker nomenclature
- Use travis job name variable in .travis.yml
- Use colorized terminal in Docker

### CODE REVIEW

File: ci/build-docker-image.sh [hunk 1]
```diff
@@ -63,7 +63,15 @@ case $image_substring in
 esac
 
 # Create tag based on branch name
-CI_DOCKER_IMAGE_NAME="$CI_DOCKER_IMAGE_NAME":$(echo "$CI_BRANCH" | tr '[:upper:]' '[:lower:]' | sed -e 's:/:_:g')
+default_branch=$(git remote show origin | grep HEAD | awk '{print $3;}')
```
Comment (MatthijsBurgh) [inline ci/build-docker-image.sh:66]: Is the `;` need after the `$3`? As we don't have it in our `__tue-git-default-branch` function

**Comment by MatthijsBurgh** [comment]
@ar13pit could we get the docker output colored?

File: ci/build-docker-image.sh (hunk 1 above)
Comment (ar13pit) [reply ci/build-docker-image.sh:66]: It is an optional syntax if there is only one command being executed. If multiple commands are executed then they must be separated by semi colons

**Comment by ar13pit** [comment]
We can have colored output only during `docker run`. I haven't found a solution for during build.

**Comment by MatthijsBurgh** [comment]
> We can have colored output only during `docker run`. I haven't found a solution for during build.

Lets do this in a separate PR.

**Comment by ar13pit** [comment]
Too late. I pushed the commit :laughing:

**Pull request merged**